react/src/base/inputs/components/SprkInputElement/SprkInputElement.js

```diff
@@ -127,6 +170,17 @@ SprkInputElement.propTypes = {
    * component in the valid or the error state.
    */
   valid: propTypes.bool,
+  /**
+   * A space-separated string of valid HTML Ids to add to the aria-describedby
+   * attribute on the Input.
+   */
+  ariaDescribedBy: propTypes.string,
```

Comment:
The descriptions are missing here. Need to be added.

Reply:
@afebbraro We didn't bother updating the descriptions for these because we're deprecating this element and adding the propTypes were to fix eslint issues. Do you think we still need to add them?